.github/workflows/ci.yaml

```
@@ -89,7 +89,7 @@ jobs:
             name: ci-artifacts
             path: artifacts-native${{ matrix.java }}.zip
   build-code-start:
```

```suggestion
  linux-build-code-start:
```
Just to make it consistent with the others...